determine_versions.py

```diff
@@ -0,0 +1,648 @@
+# License: BSD3
```

Comment:
Could `bump_recipies_versions.py` be a better name for this file? As it really update the recipe.yaml files?

Also, I've seen this yesterday https://twitter.com/condaforge/status/1647933355877445633?s=20, is there a way to do something similar here?

Reply:
> Could `bump_recipies_versions.py` be a better name for this file? As it really update the recipe.yaml files?

Sure :+1: 

> Also, I've seen this yesterday https://twitter.com/condaforge/status/1647933355877445633?s=20, is there a way to do something similar here?

That would be nice. Although likely to work on conda feedstock only? Not sure which tool allows for doing something like this but if we find it we could indeed use it in this repo (maybe @wolfv knows).